server/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wise-old-man-server",
-  "version": "2.8.10",
```

Comment:
Can you update this please? It's currently at v2.8.18

Reply:
Updated